lib/core/test/unit/Cardano/WalletSpec.hs

```diff
@@ -1284,7 +1285,14 @@ dummyTransactionLayer = TransactionLayer
         -- TODO: (ADP-957)
         let cinps' = []
         let tid = mkTxId inps' (outputsCovered cs) mempty Nothing
-        let tx = Tx tid Nothing inps' cinps' (outputsCovered cs) mempty Nothing
+        let tx = Tx tid
+                    Nothing
+                    inps'
+                    cinps'
+                    (outputsCovered cs)
+                    mempty
+                    Nothing
+                    ScriptsNotSupported
```

Comment:
Suggestion: use record construction syntax here. It's easier to guess what `Nothing` refers to if there's a field name provided!

Reply:
I'm glad you said that, because the order of inputs and collateralInputs was incorrect!

Reply:
e2dff11

Reply:
b26fcc4